web/style/transvision.css

```diff
@@ -946,3 +946,52 @@ fieldset {
         font-size: 10px;
     }
 }
+
+/* Tooltips for copy-pasting of strings */
+.clipboard {
+    float: right;
+    font-size: 200%;
+    color: #000;
+    cursor: pointer;
+}
+
+@media only screen and (max-width: 460px) {
```

Comment:
We should do this starting from 850px since we switch to the vertical layout there.